science/lal/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem    1.0
 
 name          lal
 version       6.19.0
+revision      1
```

Comment:
small detail, but the revision bump is not needed here as the port previously did not build, and you are making other changes to the port (that themselves would trigger a rebuild).